src/metaswitch/clearwater/etcd_tests/etcdcluster.py

```diff
@@ -54,3 +54,13 @@ def __del__(self):
     def delete_datadir(self):
         rmtree(self.datadir, True)
 
+    def debug(self):
+        dbg = ("Cluster servers - {}\n".format(self.servers.keys()))
+        for server in self.servers.values():
```

Comment:
Instead of giving the key and value separately - could give them together: e.g.

```python
for k in servers:
   print "Server: {} - stats {} - members {} - isAlive{}".format(
k, servers[k].getStats(),
servers[k].memberList(),
servers[k].isAlive())
```

Reply:
Wanted to pull the list of all server keys out at the top, and then have the details for each separately below. Makes it more useful i think if we want to apply this to larger clusters ever